src/service/url-expander/expander.js

```diff
@@ -125,7 +125,7 @@ export class Expander {
     let ignoringChars = false;
     let nextArgShouldBeRaw = false;
 
-    const evaluateNextLevel = () => {
+    const evaluateNextLevel = noEncode => {
```

Comment:
I always feel it's mind twisting to read an inverse boolean. 
can we change all `noEncode` to `encode`?